src/Calculator/BcMathCalculator.php

```diff
@@ -0,0 +1,188 @@
+<?php
+/**
+ * This file is part of the Money library
+ *
+ * Copyright (c) 2011-2013 Mathias Verraes
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Money\Calculator;
+
+use InvalidArgumentException;
+use Money\CalculatorInterface;
+use Money\Money;
+use Money\Number;
+
+/**
+ * Class BcMathCalculator
+ * @package Money\Calculator
```

Comment:
Please add yourself as an author (if you are OK with that) and remove the rest from here.